examples/framework/preferences/minimalEditor/js/minEditor.js

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+var minEditor = minEditor || {};
+
+(function () {
+    "use strict";
+
+    /**
+     * Primary Schema
+     * This schema defines the preference(s) edited by this preferences editor:
+     * their names, types, default values, etc.
+     */
+    minEditor.primarySchema = {
+        // the string "minEditor.autoPilot" is the 'name' of the preference
+        "minEditor.autoPilot": {
+            type: "boolean",
+            "default": false
+        }
+    };
+
+    /**
+     * Panel for the auto-pilot preference
+     */
+    fluid.defaults("minEditor.panels.autoPilot", {
+        gradeNames: ["fluid.prefs.panel"],
+
+        // the Preference Map maps the information in the primary schema to this panel
+        preferenceMap: {
+            // the key must match the name of the pref in the primary schema
+            "minEditor.autoPilot": {
+                // this key is the path into the panel's model where this preference is stored
+                "model.autoPilot": "default"
+            }
+        },
+
+        // selectors identify elements in the DOM that need to be accessed by the code
+        // in this case, the Renderer will render data into these particular elements
+        selectors: {
+            header: ".mec-autoPilot-header",
+            label: ".mec-autoPilot-label",
+            autoPilot: ".mec-autoPilot"
+        },
+
+        // the ProtoTree is basically instructions to the Renderer
+        // the keys in the prototree match the selectors above
+        protoTree: {
+            // "messagekey" refers to the keys for strings in the JSON messages file
+            header: {messagekey: "autoPilotHeader"},
+            label: {messagekey: "autoPilotLabel"},
+
+            // this string 'autoPilot' refers to the last part of the model path in the preferenceMap
```

Comment:
I wonder if you should remove the word "string" from here, as it is a bit overloaded with the previous comments. Also it's only a portion of the "string" literal that is supplied as the expression to be resolved.